templates/cicd/pipeline_cfn.yml

```diff
@@ -280,6 +280,22 @@ Resources:
               OutputArtifacts:
                 - Name: SCCheckoutArtifact
               RunOrder: 1
+        {{- else if eq .Source.ProviderName "GitHub"}}
```

Comment:
we should be able to collapse now bitbucket and github by using `isCodestarConnection`